setup.py

```diff
@@ -16,11 +16,11 @@
     # installed. This is stupid, but harmless.
     'dm_control @ https://api.github.com/repos/deepmind/dm_control/tarball/92f9913013face0468442cd0964d5973ea2089ea',  # noqa: E501
```

Comment:
I don't remember why we locked dm_control. Can you try the latest version and see if it fails?

Reply:
dm_control doesn't publish on pip/PyPI (last I checked), so the only way to retrieve it is using git. Pinning to master is a recipe for disaster, so I used the latest commit hash (at the time) instead.

Reply:
Yeah they don't publish. Should I try bringing it to the latest commit then?

Reply:
yes

Reply:
does it work?